data/content_scripts/main.js

```diff
@@ -0,0 +1,7 @@
+window.addEventListener('pageshow', function(event) {
+  var visit = new Visit(document.documentURI, document.title);
+  visit.save().then(function (response) {
+    console.log(response);
```

Comment:
do we want to console.log the response in general, or just when we have, say, some kind of debug flag set?